libbpf-rs/src/map.rs

```diff
@@ -409,6 +420,42 @@ impl Map {
     pub fn keys(&self) -> MapKeyIter {
         MapKeyIter::new(self, self.key_size())
     }
+
+    /// Create the bpf map standalone.
+    pub fn create<T: AsRef<str>>(
+        map_type: MapType,
+        name: T,
```

Comment:
in libbpf map name is optional as some (very) old kernels don't support specifying map name. So I wonder if this should be an Option[T]?

Reply:
You're right, `Option<T>` will be better.